src/pages/edge/adobe-journey-optimizer-decisioning/review-simulate.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Adobe Experience Platform Assurance Review & Simulate
+description: This is a feature of the Adobe Experience Platform Assurance which allows the users to review the content sent to devices for each request and simulate different experiences on the same devices in real time.
+keywords:
+- Adobe Experience Platform Assurance
+- Review & Simulate Feature Introduction
+---
+
+# Adobe Experience Platform Assurance Review & Simulate
+
+This is a feature of the Adobe Experience Platform Assurance which allows the users to review the content sent to devices for each request and simulate different experiences on the same devices in real time.
+
+## Review Requests and Results
+
+1. [Setup and connect](./assurance-for-decisioning.md) ot an Assurance session.
+2. To review the personlization requests and results navigate to ‘Review & Simulate’ tab.
+   Make sure that at least one decision scope has been set up in the application to be able review them. 
+
+3. In the review tab you can see all personalization requests in the session. If you do not see any request, trigger an update request from the application.
+
+   ![](./assets/review-simulate/assurance-review-tab.png)
+
+4. Select any request to see more information about it in the expanded right panel. To see the preview the result, click the ‘Returned Content’ button available in the right panel. 
+
+   ![](./assets/review-simulate/assurance-result-preview.png)
+
+
+## Simulate Different Results
+
+The result of each personalization request is customized for each user based on several deciding factors as configured. A unique experience is served for each user. Simulate allows the user to view all available experiences on the device in real time.  
```

Comment:
Simulate allows the user to view a different experience then the one they currently serviced.